pkg/controller/servingcert/controller/secret_updating_controller.go

```diff
@@ -210,8 +209,7 @@ func (sc *serviceServingCertUpdateController) ensureSecretData(service *v1.Servi
 	}
 
 	// set the owning-component unless someone else has claimed it.
-	if !update && len(secretCopy.Annotations[apiannotations.OpenShiftComponent]) == 0 {
-		secretCopy.Annotations[apiannotations.OpenShiftComponent] = api.OwningJiraComponent
+	if !update && setSecretOwnerDescription(secretCopy, service) {
```

Comment:
```suggestion
	needsOwnerUpdate := setSecretOwnerDescription(secretCopy, service) 
	if !update && needsOwnerUpdate {
```

slightly easier for reviewer to read.

Reply:
Done